caluma/caluma_workflow/serializers.py

```diff
@@ -448,3 +510,63 @@ class Meta:
             "meta",
             "context",
         )
+
+
+class SuspendWorkItemSerializer(ContextModelSerializer):
+    id = serializers.GlobalIDField()
+
+    def validate(self, data):
+        try:
+            domain_logic.SuspendWorkItemLogic.validate_for_suspend(self.instance)
+        except ValidationError as e:
+            raise exceptions.ValidationError(str(e))
+
+        return super().validate(data)
+
+    def update(self, work_item, validated_data):
```

Comment:
Missing `@transaction.atomic`

Reply:
True! I [fixed the missing transactions](c7963005fbfe1fabeded03ae8304a9cda6bc1239) in the whole file. Since we always trigger an event that could potentially do DB changes it should be used in every `update` or `create`.